From 608a970914a3725a61f65d61d1697c019a006be6 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sat, 15 Aug 2020 19:01:20 +0200
Subject: [PATCH 3/3] Solus tweaks to the default config

- Decrease desktop icon sizes
- Switch to launchermenu
- Move showdesktop between systemtray and the clock
---
 containments/desktop/package/contents/config/main.xml     | 2 +-
 .../contents/layout.js                                    | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 5703999..8c02e38 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="labelWidth" type="Int">
       <label>The width of the grid cells when using icon view. 0 = Narrow, 1 = Medium, 2 = Wide</label>
diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index 048671e..cf48b98 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -27,9 +27,9 @@ if (freeEdges["bottom"] == true) {
 
 panel.height = gridUnit * 2
 
-var kickoff = panel.addWidget("org.kde.plasma.kickoff")
-kickoff.currentConfigGroup = ["Shortcuts"]
-kickoff.writeConfig("global", "Alt+F1")
+var kicker = panel.addWidget("org.kde.plasma.kicker")
+kicker.currentConfigGroup = ["Shortcuts"]
+kicker.writeConfig("global", "Alt+F1")
 
 //panel.addWidget("org.kde.plasma.showActivityManager")
 panel.addWidget("org.kde.plasma.pager")
@@ -79,6 +79,6 @@ if (langIds.indexOf(languageId) != -1) {
 }
 
 panel.addWidget("org.kde.plasma.systemtray")
-panel.addWidget("org.kde.plasma.digitalclock")
 panel.addWidget("org.kde.plasma.showdesktop")
+panel.addWidget("org.kde.plasma.digitalclock")
 
-- 
2.27.0

